data/kernels/blendop.cl

```diff
@@ -580,49 +583,47 @@ blendop_Lab (__read_only image2d_t in_a, __read_only image2d_t in_b, __read_only
       o =  clamp((a * opacity) + (b * (1.0f - opacity)), min, max);
       break;
 
-    case DEVELOP_BLEND_BOUNDED:
-    case DEVELOP_BLEND_NORMAL:
-      o =  clamp((a * (1.0f - opacity)) + (b * opacity), min, max);
-      break;
-
     case DEVELOP_BLEND_LAB_LIGHTNESS:
     case DEVELOP_BLEND_LAB_L:
-      o.x = (a.x * (1.0f - opacity)) + (b.x * opacity);
-      o.y = a.y;
-      o.z = a.z;
+      o.x = clamp((a.x * (1.0f - opacity)) + (b.x * opacity), min.x, max.x);
```

Comment:
simple alpha blending like that should not clip

Reply:
Code is kept as is for Lab/raw/display RGB blending modes. Scene referred blending mode does not clip.